Frontend/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:20.7.0-alpine AS build
+WORKDIR /app
+COPY package*.json ./
+RUN npm install -g npm@10.2.5
+RUN npm ic
+COPY . .
+RUN npm run build
+FROM nginx:stable
+COPY --from=build /app/dist/frontend /usr/share/nginx/html
+EXPOSE 80
```

Comment:
We are going to put this behind a reverse proxy so we don't want to expose port 80 directly, let's go with a higher port number.

Reply:
How about 4200 then? Is that good?

Reply:
that works!